extensions/common/iam/identity-trust/identity-trust-sts/identity-trust-sts-accounts-api/src/main/resources/sts-accounts-api-version.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "version": "1.0.0",
+    "urlPath": "/v1alpha",
+    "lastUpdated": "2024-09-24T14:14:00Z"
```

Comment:
`maturity` is missing

Reply:
i thought we can't set `maturity`, unless it is `stable`?

Reply:
maybe you're right... I don't remember :smiling_face_with_tear: 

EDIT: you're right